draft-ietf-tls-esni.md

```diff
@@ -668,15 +684,15 @@ the new EncodedClientHelloInner value as a second message with the previous
 HPKE context:
 
 ~~~
-    payload = context.Seal(ClientHelloOuterAAD,
-                           EncodedClientHelloInner)
+    payload = context.Seal(ech_aad, EncodedClientHelloInner)
 ~~~
 
-ClientHelloOuterAAD is computed as described in {{authenticating-outer}}, but
-again using the second ClientHelloOuter. Note an HPKE context maintains a
-sequence number, so this operation internally uses a fresh nonce for the AEAD
-operation. Reusing the HPKE context avoids an attack described in
-{{flow-hrr-hijack}}.
+ech_aad is computed from ClientECH and ClientHelloOuterAAD as described in
+{{real-ech}}. ClientHelloOuterAAD is computed as described in
+{{authenticating-outer}}, but again using the second ClientHelloOuter. Note an
```

Comment:
ClientHelloOuterAAD is only used to compute ech_aad anyway. Seems the new text would be better off in {{authenticating-outer}} and then we only need to cite one section and variable name as before.